core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -639,20 +639,6 @@ private[spark] object SparkConf extends Logging {
    */
   private val deprecatedConfigs: Map[String, DeprecatedConfig] = {
     val configs = Seq(
-      DeprecatedConfig("spark.cache.class", "0.8",
```

Comment:
do we need to remove these? they are warnings for users if they set the wrong config right

Reply:
Yeah I can add them back. Wasn't sure whether they are still valuable or just old.